metadata-ingestion/src/datahub/ingestion/source/redshift/usage.py

```diff
@@ -325,6 +325,10 @@ def _gen_operation_aspect_workunits_from_access_events(
             ):
                 continue
 
+            if not self._should_process_event(event, all_tables=all_tables):
+                self.report.num_operational_stats_skipped += 1
+                continue
```

Comment:
Think is checked twice, once here and once in `_gen_access_events_from_history_query`